litho-it/src/test/java/com/facebook/litho/widget/TestRecyclerView.java

```diff
@@ -0,0 +1,26 @@
+package com.facebook.litho.widget;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import java.util.ArrayList;
+import java.util.List;
+
+/** A test {@link RecyclerView} class used for unit testing. */
+public class TestRecyclerView extends RecyclerView {
+
+  private final List<Runnable> postAnimationRunnableList = new ArrayList<>();
+
+  public TestRecyclerView(@NonNull Context context) {
+    super(context);
+  }
+
+  @Override
+  public void postOnAnimation(Runnable r) {
+    postAnimationRunnableList.add(r);
+  }
+
+  public List<Runnable> getPostAnimationRunnableList() {
```

Comment:
Do we need actual collection or just `count` will be enough, as in `TestLithoRecyclerView`?

Reply:
Yes, we need the complete collection since we want to verify the all the runnable instances in postAnimationRunnableList is same.